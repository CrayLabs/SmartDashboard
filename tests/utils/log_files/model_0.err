Traceback (most recent call last):
  File "/lus/cls01029/mellis/cray_labs/SmartSim/tests/test_configs/bad.py", line 42, in <module>
    divide_by_zero(args.time)
  File "/lus/cls01029/mellis/cray_labs/SmartSim/tests/test_configs/bad.py", line 34, in divide_by_zero
    print(1 / 0)
ZeroDivisionError: division by zero