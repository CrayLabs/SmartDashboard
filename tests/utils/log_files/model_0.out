SmartRedis Library@16-04-21:WARNING: Environment variable SR_LOG_FILE is not set. Defaulting to stdout
SmartRedis Library@16-04-21:WARNING: Environment variable SR_LOG_LEVEL is not set. Defaulting to INFO
Freq (hz) = 0.1
Sleep time = 10.0
Number of messages = 10
